requirements.txt

```diff
@@ -5,14 +5,18 @@ flask-jsonpify
 flask-oauthlib
 pyyaml
 boto
-raven==6.0.0
 requests
 pyjwt
 sqlalchemy
 psycopg2
 cryptography
 elasticsearch>=1.0.0,<2.0.0
-os-package-registry>=0.0.12
+os-package-registry>=0.0.13
+datapackage-pipelines-fiscal>=1.0.7
+datapackage-pipelines-aws
+datapackage-pipelines[speedup]
```

Comment:
I've not used that square-brackets syntax before. Learned something new!

Reply:
👍